### PR TITLE
add object detection doc

### PR DESCRIPTION
**Pull request by dding3**

### CODE REVIEW

File: docs/docs/APIGuide/Models/object-detection.md [hunk 1]
```diff
@@ -1 +1,122 @@
+# Analytics Zoo Object Detection API
 
+Analytics Zoo provides a collection of pre-trained models for Object Detection. These models can be used for out-of-the-box inference if you are interested in categories already in the corresponding datasets. According to the business scenarios, users can embed the models locally, distributedly in Spark such as Apache Storm and Apache Flink.
```
Comment (hhbyyh) [inline docs/docs/APIGuide/Models/object-detection.md]: "users can embed the models locally, distributedly in Spark such as Apache Storm and Apache Flink." --> 

User can run the inference as local program without Spark Context, or in a distributed environment such like Apache Spark, Apache Storm or Apache Flink.

File: docs/docs/APIGuide/Models/object-detection.md [hunk 2]
```diff
@@ -1 +1,122 @@
...
+
+## Model Load
+
+Use `ObjectDetector.loadModel`(in Scala) or `ObjectDetector.load_model` (in Python) to load an pre-trained Analytics Zoo model or third-party(BigDL) model. We just need to specify the model path and optionally weight path if exists where we previously saved the model.
```
Comment (hhbyyh) [inline docs/docs/APIGuide/Models/object-detection.md]: Just to confirm, can we load BigDL model?

Comment (dding3) [reply docs/docs/APIGuide/Models/object-detection.md]: Yes, we support it now.

**Comment by dding3** [comment]
Jekins pass. http://172.168.2.101:8080/job/ZOO-PR-Validation/487/

**Pull request merged**